service/build.gradle

```diff
@@ -34,6 +34,7 @@ dependencies {
     implementation 'bio.terra:datarepo-client:2.13.0-SNAPSHOT'
     implementation 'org.postgresql:postgresql:42.7.2'
     implementation 'jakarta.ws.rs:jakarta.ws.rs-api'
+    implementation 'bio.terra:externalcreds-client-resttemplate:1.12.0-20240229.204210-1'
```

Comment:
nit: since you have a method calling `oauthApi.getProviderAccessToken` in EcmService (even though it might not be used until later PR), it might be good to update the client to 1.15 to avoid the `500 error no static resource /api/oidc..` that we were seeing earlier.